### PR TITLE
getting started integration tests

### PR DESCRIPTION
**Pull request by VicVerevita**
I have picked up on the #777 issue. I saw that there was already a pull request tackling the integration tests for sqlite, so I started the postgresql ones.

To run the tests, I used the `test_all` script.

### CODE REVIEW

**Comment by VicVerevita** [comment]
This is my first ever open-source pr, so if there are any good practices that I can add to the tests/description, I am glad to take reviews on that as well.

File: examples/mysql/getting_started_step_1/Cargo.toml [hunk 1]
```diff
@@ -7,6 +7,7 @@ publish = false
 [dependencies]
 diesel = { version = "2.1.0", path = "../../../diesel", features = ["mysql"] }
 dotenvy = "0.15"
+assert_cmd = "2.0.14"
```
Comment (weiznich) [inline examples/mysql/getting_started_step_1/Cargo.toml:10]: This should be a `dev-dependency` as we only use that in tests

File: examples/mysql/getting_started_step_2/Cargo.toml [hunk 2]
```diff
@@ -7,6 +7,7 @@ publish = false
 [dependencies]
 diesel = { version = "2.1.0", path = "../../../diesel", features = ["mysql"] }
 dotenvy = "0.15"
+assert_cmd = "2.0.14"
```
Comment (weiznich) [inline examples/mysql/getting_started_step_2/Cargo.toml:10]: This should be a `dev-dependency` as we only use that in tests

File: examples/mysql/getting_started_step_3/Cargo.toml [hunk 3]
```diff
@@ -7,6 +7,7 @@ publish = false
 [dependencies]
 diesel = { version = "2.1.0", path = "../../../diesel", features = ["mysql"] }
 dotenvy = "0.15"
+assert_cmd = "2.0.14"
```
Comment (weiznich) [inline examples/mysql/getting_started_step_3/Cargo.toml:10]: This should be a `dev-dependency` as we only use that in tests

File: examples/postgres/getting_started_step_1/Cargo.toml [hunk 4]
```diff
@@ -7,6 +7,7 @@ publish = false
 [dependencies]
 diesel = { version = "2.1.0", path = "../../../diesel", features = ["postgres"] }
 dotenvy = "0.15"
+assert_cmd = "2.0.14"
```
Comment (weiznich) [inline examples/postgres/getting_started_step_1/Cargo.toml:10]: This should be a `dev-dependency` as we only use that in tests

File: examples/postgres/getting_started_step_2/Cargo.toml [hunk 5]
```diff
@@ -7,6 +7,7 @@ publish = false
 [dependencies]
 diesel = { version = "2.1.0", path = "../../../diesel", features = ["postgres"] }
 dotenvy = "0.15"
+assert_cmd = "2.0.14"
```
Comment (weiznich) [inline examples/postgres/getting_started_step_2/Cargo.toml:10]: This should be a `dev-dependency` as we only use that in tests

File: examples/postgres/getting_started_step_3/Cargo.toml [hunk 6]
```diff
@@ -7,6 +7,7 @@ publish = false
 [dependencies]
 diesel = { version = "2.1.0", path = "../../../diesel", features = ["postgres"] }
 dotenvy = "0.15"
+assert_cmd = "2.0.14"
```
Comment (weiznich) [inline examples/postgres/getting_started_step_3/Cargo.toml:10]: This should be a `dev-dependency` as we only use that in tests

File: examples/sqlite/getting_started_step_1/Cargo.toml [hunk 7]
```diff
@@ -10,6 +10,7 @@ publish = false
 diesel = { version = "2.1.0", path = "../../../diesel", features = ["sqlite"] }
 dotenvy = "0.15"
 libsqlite3-sys = { version = "0.28.0", features = ["bundled"] }
+assert_cmd = "2.0.14"
```
Comment (weiznich) [inline examples/sqlite/getting_started_step_1/Cargo.toml:13]: This should be a `dev-dependency` as we only use that in tests

File: examples/sqlite/getting_started_step_2/Cargo.toml [hunk 8]
```diff
@@ -9,6 +9,7 @@ publish = false
 diesel = { version = "2.1.0", path = "../../../diesel", features = ["sqlite", "returning_clauses_for_sqlite_3_35"] }
 dotenvy = "0.15"
 libsqlite3-sys = { version = "0.28.0", features = ["bundled"] }
+assert_cmd = "2.0.14"
```
Comment (weiznich) [inline examples/sqlite/getting_started_step_2/Cargo.toml:12]: This should be a `dev-dependency` as we only use that in tests

File: examples/sqlite/getting_started_step_3/Cargo.toml [hunk 9]
```diff
@@ -9,6 +9,7 @@ publish = false
 diesel = { version = "2.1.0", path = "../../../diesel", features = ["sqlite", "returning_clauses_for_sqlite_3_35"] }
 dotenvy = "0.15"
 libsqlite3-sys = { version = "0.28.0", features = ["bundled"] }
+assert_cmd = "2.0.14"
```
Comment (weiznich) [inline examples/sqlite/getting_started_step_3/Cargo.toml:12]: This should be a `dev-dependency` as we only use that in tests

**Review by weiznich — COMMENTED** [review]
Thanks for working on this :+1:

Looks good for me beside the fact the the trycmd dependency should be a dev-dependency.

**Review by weiznich — APPROVED** [review]

**Comment by weiznich** [comment]
Thanks again for working on this. This is good work for your first PR :tada:

**Pull request merged**